### ISSUE TITLE
# [Global] Input borders feature low contrast

### ISSUE THREAD

**Issue by svinkle** [Shopify/dawn#1149]
## Issue summary

`input` and `textarea` controls feature low contrast borders.

Search:

- Border: `#d7d4d2`
- Background: `#faf7f2`
- Contrast ratio: `1.38:1`

Contact form:

- Border: `#cfccc6`
- Background: `#faf7f3`
- Contrast ratio: `1.5:1`

Footer email:

- Border: `#d8c4c3`
- Background: `#f0fdad`
- Contrast ratio: `1.53:1`

<details>
  <summary>Screenshots</summary>

  <img width="591" alt="Search form with arrows pointing to input background and border." src="https://user-images.githubusercontent.com/1392632/148973304-850a0f4b-f51f-4ecf-8451-c588faecc925.png">

  <img width="830" alt="Contact form with arrows pointing to input background and border." src="https://user-images.githubusercontent.com/1392632/148973348-2bcddeab-a8c3-4210-942b-705f9775cb2e.png">

  <img width="632" alt="l form with arrows pointing to input background and border." src="https://user-images.githubusercontent.com/1392632/148973387-a728f779-ba8c-4c44-883d-7b8f0f02c23c.png">

</details>

<!-- ### Current code

#### HTML

```html

``` -->

## Steps to reproduce

1. Check the [color contrast of the Search input](https://contrast-ratio.com/#%23d7d4d2-on-%23faf7f2).
2. Check the [color contrast of the Contact form input](https://contrast-ratio.com/#%23cfccc6-on-%23faf7f3).
3. Check the [color contrast of the Footer email input](https://contrast-ratio.com/#%23d8c4c3-on-%23f0fdad).

## Behavior

### Expected

- Color contrast to meet AA requirements.

### Actual

- Color contrast does not meet AA requirements.

## Recommendation

Adjust the value of the border color to increase the color contrast.

<!-- ### Recommended code

#### HTML

```html

``` -->

<!-- #### Diff

```diff
- line removed
+ line added
``` -->

## Specifications

- **Browser:** Any
- **Dawn version:** 2.5.0
- **Component:** Global
- **WCAG Principal:** Perceivable
- **WCAG SC:** [1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
- **Severity:** High
- **Effort:** Medium

**Comment by ludoboludo** [comment]
I was looking at the AA requirements and for `non-text contrast` and the contrast ratio should be `3:1`.
The screenshots seem to be specific to the theme `Sense`. Changing the input border opacity to `55%` instead of `15%` seem to solve the issue. I'm creating a PR right now.
